extensions/panache/hibernate-orm-panache/runtime/src/main/java/io/quarkus/hibernate/orm/panache/runtime/JpaOperations.java

```diff
@@ -135,6 +142,10 @@ static String createFindQuery(Class<?> entityClass, String query, int paramCount
         return "FROM " + getEntityName(entityClass) + " WHERE " + query;
     }
 
+    static boolean isNamedQuery(String query) {
+        return query.charAt(0) == '#';
```

Comment:
Given that we're testing for this before we check for `isEmpty` this can throw.

Reply:
Fixed